lib/Pinto/Action/Pull.pm

```diff
@@ -45,7 +45,7 @@ with qw( Pinto::Role::Committable Pinto::Role::Puller );
 sub BUILD {
     my ($self) = @_;
 
-    $self->stack->assert_not_locked;
+    $self->stack->assert_not_locked unless $self->dry_run;
```

Comment:
Could this be moved to the Committable role so that all Actions will get the behavior? That's where the `dry_run` attribute is defined, so it also seems like the obvious place to put this.

But I must have put it here for a reason. Or maybe I just wasn't thinking.